api/calculate_frequencies.py

```diff
@@ -117,19 +64,85 @@ def calculate_frequencies3(genotypedata, alleles_definitions):
 
 	ncol = max(freq_length)
 
+	# create matrix with frequency values from frequencies list
+	freqmatrix = _create_frequencyMatrix(nloci, ncol, frequencies)
+
+	# final result
+	ret = _pack_result(freq_length, freqmatrix, variability)
+
+	return ret
+
+def _getLocinames(genotypedata: pd.DataFrame):
```

Comment:
Both calculate_frequencies and define_alleles use this function; can you move it into `recrudescence_utils` and import it into both files instead?

Reply:
Also, add a brief docstring/method comment to this explaining what it does (not everyone knows what locinames are, so explain that)